package.json

```diff
@@ -265,6 +265,9 @@
     "**/globule/minimatch": "^3.1.2",
     "ansi-regex": "^5.0.1",
     "testcafe/**/jsonwebtoken": "^9.0.0",
+    "json5": "^2.2.2",
```

Comment:
I believe we don't need this line. 
It forces using v2 for all transitive dependencies (even those ones that require v1) and causes removing json5 v1 from lock file:
https://github.com/okta/okta-signin-widget/pull/3602/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL13407